src/routes/dashboard/lab/+page.svelte

```diff
@@ -18,17 +42,60 @@
         </li>
       {/each}
     </ul>
+    {#if faculty.length > 0}
+      <h3 class="h3">Faculty</h3>
+      <ul class="space-y-1">
+        {#each faculty as user (user.email)}
+          <li
+            class="preset-filled-surface-100-900 hover:preset-filled-surface-200-800 rounded-md p-2 transition-colors duration-150"
+          >
+            <Member {user} />
+          </li>
+        {/each}
+      </ul>
+    {/if}
   </nav>
   <nav class="list-nav space-y-2">
     <h3 class="h3">Members</h3>
-    <ul class="space-y-1">
-      {#each members as user (user.email)}
-        <li
-          class="preset-filled-surface-100-900 hover:preset-filled-surface-200-800 rounded-md p-2 transition-colors duration-150"
-        >
-          <Member {user} />
-        </li>
-      {/each}
-    </ul>
+    <div class="space-y-1">
+      <Accordion multiple collapsible>
+        {#each membersByDraft as { draftId, memberUsers } (draftId)}
+          {@const draft = drafts.find(draft => Number(draft.id) === draftId)}
+          {#if typeof draft !== 'undefined'}
+            {@const start = format(draft.activePeriodStart, 'PPPpp')}
+            {@const end = format(draft.activePeriodEnd, 'PPPpp')}
+            <Accordion.Item value="draft-{draftId}">
+              {#snippet control()}
+                <div class="flex flex-col">
+                  <span class="h4">Draft {draftId}</span>
+                  <small>
+                    {#if draft.activePeriodEnd !== null}
+                      <time datetime={draft.activePeriodStart.toISOString()}>{start}</time>
+                      to
+                      <time datetime={draft.activePeriodEnd.toISOString()}>{end}</time>
+                    {:else}
+                      Ongoing since <time datetime={draft.activePeriodStart.toISOString()}
+                        >{start}</time
+                      >
+                    {/if}
```

Comment:
Let's invert the condition here [so that the logic is affirmative](https://dev.to/somedood/please-dont-write-confusing-conditionals-2n32).